db/migrate/20150114024350_create_course_announcements.rb

```diff
@@ -0,0 +1,14 @@
+class CreateCourseAnnouncements < ActiveRecord::Migration
+  def change
+    create_table :course_announcements do |t|
+      t.references :course, null: false
+      t.string :title, null: false
+      t.text :content
```

Comment:
I prefer content, description is like describe something.  So I use
description for course model

On Wed, Jan 14, 2015 at 11:04 AM, Joel Low <notifications@github.com> wrote:

> In db/migrate/20150114024350_create_course_announcements.rb
> <https://github.com/Coursemology/coursemology2/pull/49#discussion-diff-22914230>
> :
>
> > @@ -0,0 +1,14 @@
> > +class CreateCourseAnnouncements < ActiveRecord::Migration
> > +  def change
> > +    create_table :course_announcements do |t|
> > +      t.references :course, null: false
> > +      t.string :title, null: false
> > +      t.text :content
>
> is content better or description better?
> @wangqiang1208 <https://github.com/wangqiang1208> @jsyeo
> <https://github.com/jsyeo> @weiqingtoh <https://github.com/weiqingtoh>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/49/files#r22914230>.
>